docs/src/literate-tutorials/reactive_surface.jl

```diff
@@ -0,0 +1,259 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if isdefined(Main, :is_ci) #hide
+    IS_CI = Main.is_ci     #hide
+else                       #hide
+    IS_CI = false          #hide
+end                        #hide
+# # [Reactive Surface](@id tutorial-reactive-surface)
+#
+# ![](reactive_surface.gif)
```

Comment:
This file is missing. Is it large? In that case we can put it on the gh-pages branch directly like we do with some other files and then download during the build.

Reply:
Here it is: https://private-user-images.githubusercontent.com/9196588/239651993-a2975c20-36c0-4a6b-979d-a4a0553223ec.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcxNjA4MTgsIm5iZiI6MTcxNzE2MDUxOCwicGF0aCI6Ii85MTk2NTg4LzIzOTY1MTk5My1hMjk3NWMyMC0zNmMwLTRhNmItOTc5ZC1hNGEwNTUzMjIzZWMuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUzMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MzFUMTMwMTU4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGE5OWZmOTVmNTNkMzM4M2M3MzA3MWNiMGEwYmJmMjgzNjdiOTI4NDJlZWNjNjA4OWE5NzZlM2IxM2ExNTVmYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.DZLlv5Kw_qM3O2XpvhW-dnci5B2e8pnkjWvnKp_PGx4 
Can you add it to the branch? I will then update the downloader.

Reply:
Link doesn't seem to work. Is the the same as in the first post?

Reply:
Yes its the same as here https://github.com/notifications#issue-1717915883